dotnet-authserver/src/TeacherIdentity.AuthServer/DqtApiClient.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.Net.Http.Headers;
+using TeacherIdentity.AuthServer.Models;
+
+namespace TeacherIdentity.AuthServer;
+
+public class DqtApiClient : IDqtApiClient
+{
+    private IConfiguration _configuration;
+
+    public DqtApiClient(IConfiguration configuration)
+    {
+        _configuration = configuration;
+    }
+
+    private HttpClient GetHttpClient()
```

Comment:
Preferable to get the `HttpClient` injected into the constructor (see https://docs.microsoft.com/en-us/dotnet/api/microsoft.extensions.dependencyinjection.httpclientfactoryservicecollectionextensions.addhttpclient?view=dotnet-plat-ext-6.0)